ci-operator/step-registry/openshift/e2e/aws/etcd-scaling/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- deads2k
+- dgoodwin
+- dusk125
+- elbehery
+- emilym1
+- hasbro17
+- tjungblu
```

Comment:
also add @wking, I'm sure he also will have to go and plumb some stuff there once in a while.

Reply:
feel free to add update in your follow-up PR, let's get this through first ;)